apps/products-feed/src/modules/category-mapping/ui/instructions.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link, Paper, Typography } from "@material-ui/core";
+import { actions, useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { makeStyles } from "@saleor/macaw-ui";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    instructionsContainer: {
+      padding: 15,
+    },
+  };
+});
+
+export const Instructions = () => {
+  const styles = useStyles();
+
+  const { appBridge } = useAppBridge();
+
+  return (
+    <Paper elevation={0} className={styles.instructionsContainer}>
+      <Typography paragraph variant="h4">
+        Using the feed
+      </Typography>
+      <Typography paragraph>
+        Configure your Google Merchant account to schedule fetches of the feed. Instructions can be
+        found{" "}
+        <Link
+          href="#"
```

Comment:
Shouldn't the `href` be equal to the address we redirect to in line 31? For accessibility purposes.